src/styles/group-list.scss

```diff
@@ -72,6 +72,11 @@ $group-list-spacing-below: 50px;
       font-weight: bold;
     }
   }
+
+  .h-icon-third-party {
```

Comment:
The `h-icon-` classes are all part of the generated CSS for our icomoon icon font. This class should follow the usual `component__element` syntax.